### DISCUSSION TITLE
Using Chromebook (Linux): USB does not appear as an option in the NW menu

### DISCUSSION THREAD

**Discussion by Valdus92** [vkbo/novelWriter#1471] [Help]
I realize this may be a problem with linux, but when I try to open a project on my usb it does not appear in the nw menu. I even set the usb to share with linux. 

Anyone else?

**Comment by vkbo** [comment]
Which menu is this?

**Reply by Valdus92** [reply]
The menu for choosing a project. It does not recognize the usb.

**Reply by vkbo** [reply]
You mean the list or recently opened projects? It only shows projects you have opened before.

It's really hard to understand what you mean by "menu" since there are no menus at all involved here. So you obviously mean something else, and I'm just trying to figure it out.

**Comment by Valdus92** [comment]
When I try to find my file on my usb it does not appear on the following screen. Is this a linux problem or something else?
![Screenshot 2023-07-11 12 49 27](https://github.com/vkbo/novelWriter/assets/26496870/ded16aea-d81e-4b50-81a9-56fe08c33b5a)

**Reply by vkbo** [reply]
The file dialog is not provided by novelWriter. It uses the standard Qt implementation, which will either open whatever your desktop manager or OS provides, or it will fall back to Qt's own implementation. I'm not sure what this is, but it looks to me to be a Qt dialog. In either case, novelWriter is not in control of it. I cannot say if this is a Linux, desktop manager, or Qt issue.

I cannot find any settings that allows novelWriter to control this either.

**Comment by HeyMyian** [comment]
You could try to navigate to the USB stick in the file manager where the stick presumably is detected (outside of novelWriter) and "Open with" the project's `.nwx` file with the novelWriter executable. After this is done once, the project should/could appear in the list of recently opened projects?

**Comment by rumplestilzken** [comment]
> When I try to find my file on my usb it does not appear on the following screen. Is this a linux problem or something else?
> ![Screenshot 2023-07-11 12 49 27](https://github.com/vkbo/novelWriter/assets/26496870/ded16aea-d81e-4b50-81a9-56fe08c33b5a)
> 

Chromebook Linux VM doesn't have access to folders you don't specifically give it access to. You need to right click the drive in ChromeOS and select share with Linux.
